sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+<Enumeration>x,y,z</Enumeration>
+<SALTelemetry>
+ <Subsystem>ESS</Subsystem>
+ <Version>3.5.0</Version>
+ <Author>salgenerator</Author>
+ <EFDB_Topic>ESS_airTurbulence</EFDB_Topic>
+      <Description>Air Turbulence Instrument - CSAT3B (Tri-Axial) Serial to Ethernet Converter</Description>
+      <item>
+          <EFDB_Name>ux</EFDB_Name>
+	  <Description>x-axis wind speed</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>m/s</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>uy</EFDB_Name>
+	  <Description>y-axis wind speed</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>m/s</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>uz</EFDB_Name>
+	  <Description>z-axis wind speed</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>m/s</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>ts</EFDB_Name>
+	  <Description>Sonic temperature</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>unitless</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>diagWord</EFDB_Name>
+	  <Description>Diagnostic word</Description>
+          <IDL_Type>short</IDL_Type>
+          <Units>unitless</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>recordCounter</EFDB_Name>
+	  <Description>Record counter (0-63)</Description>
+          <IDL_Type>short</IDL_Type>
+          <Units>unitless</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+</SALTelemetry>
+<SALTelemetry>
+ <Subsystem>ESS</Subsystem>
+ <Version>3.5.0</Version>
+ <Author>salgenerator</Author>
+ <EFDB_Topic>ESS_airFlow</EFDB_Topic>
+      <Description>AirFlow Sensor - Windsonic1-L (Bi-axial) Serial to Ethernet Converter</Description>
+      <item>
+          <EFDB_Name>direction</EFDB_Name>
+	  <Description>Wind direction Telemetry</Description>
+          <IDL_Type>short</IDL_Type>
+          <Units>unitless</Units>

```

Comment:
deg? rad? I expect something like this unit..

Reply:
I will fix this after feedback from vendor, type short does not seem to fit anything obvious